controller/array_action/array_mediator_ds8k.py

```diff
@@ -137,13 +137,16 @@ def _connect(self):
                 'Failed to connect to DS8K array {}, reason is {}'.format(
                     self.service_address,
                     e.details
-                    )
                 )
+            )
             raise ConnectionError()
 
     def disconnect(self):
         pass
 
+    def is_active(self):
+        return True
```

Comment:
can't we perform some external i/o  (e.g. `.get_systems()`) and except an error (e.g. `pyds8k.exceptions.ConnectionError `) to tell if it is active?

Reply:
Possible but since I don't know what is returned for example when when token is up-to-date I prefer to handle any exception this way.